PyNifly/__init__.py

```diff
@@ -3437,7 +3436,10 @@ def extract_mesh_data(self, obj, arma, target_key):
             obj.active_shape_key_index = 0
             bpy.ops.object.mode_set(mode = 'EDIT')
             bpy.ops.object.mode_set(mode = 'OBJECT')
-                
+            if self.flag_set(pynFlags.export_modifiers):
+                depsgraph = bpy.context.evaluated_depsgraph_get()
+                obj = obj.evaluated_get(depsgraph)            
```

Comment:
What are the implications of this, and do you need to do it anyway? You're assigning the result of evaluated_get back to obj, which changes obj--doesn't it? Instead couldn't you use it without reassigning it, e.g. assign it to obj1 and then set editmesh = obj1.data.

I've never used these calls so I don't know the details of how they work.

Reply:
The assigned obj from depsgraph evaluation It won't affect to obj outside the scope. So original mesh data won't get changed.